From: Sam Ravnborg <sam@ravnborg.org>
Date: Tue, 18 Sep 2007 19:12:26 +0000 (+0200)
Subject: kconfig: make comments stand out in menuconfig
X-Git-Tag: v2.6.24-rc1~826^2~24
X-Git-Url: http://git.kernel.org/?p=linux%2Fkernel%2Fgit%2Ftorvalds%2Flinux-2.6.git;a=commitdiff_plain;h=48874077ddd6c0c444758059af2cf77c10204ece

kconfig: make comments stand out in menuconfig

MatÄj Laitl <strohel@gmail.com> noticed that there was no way
to distingush between comments and un-selectable menu lines.
This patch marks comments with *** comment ***

Cc: MatÄj Laitl <strohel@gmail.com>
Signed-off-by: Sam Ravnborg <sam@ravnborg.org>

reversed patch

Signed-off-by: Marc Kleine-Budde <mkl@pengutronix.de>

---
 mconf.c |    8 --------
 1 file changed, 8 deletions(-)

Index: kconfig/mconf.c
===================================================================
--- kconfig.orig/mconf.c
+++ kconfig/mconf.c
@@ -439,14 +439,6 @@ static void build_conf(struct menu *menu
 				if (single_menu_mode && menu->data)
 					goto conf_childs;
 				return;
-			case P_COMMENT:
-				if (prompt) {
-					child_count++;
-					item_make("   %*c*** %s ***", indent + 1, ' ', _(prompt));
-					item_set_tag(':');
-					item_set_data(menu);
-				}
-				break;
 			default:
 				if (prompt) {
 					child_count++;
